frontend/src/routes/(unauthenticated)/sandbox/i18n/ssr-only/+page.ts

```diff
@@ -0,0 +1,10 @@
+import type { PageLoadEvent } from './$types';
+
+export const csr = false;
```

Comment:
minor thing, but on this page the loading indicator never goes away, probably because of this.

Reply:
Yeah. I can't think of an easy way to fix that. 🤷